docs/cubeviz/displaycubes.rst

```diff
@@ -110,6 +110,19 @@ of the selected area, but will expand or limit the y range to preserve the aspec
 ratio of the image. The 1D spectrum viewer allows panning and zooming along a single
 axis as in Specviz.
 
+.. _cubeviz-spectrum-at-spaxel:
+
+Spectrum At Spaxel
+==================
+
+This tool allows a user to select one pixel in the image viewer and visualize the spectrum
+at that point in the spectrum viewer. The selected pixel will be highlighted in the image viewer
+and the spectrum will have the same color. If there are multiple data selected in the image viewer
```

Comment:
I don't think it makes sense to even allow this on clicks outside of flux-viewer, since the spectrum-viewer only corresponds to flux-viewer. It gets too confusing to see spaxel from, say, uncert-viewer shown but we don't have a way to collapse anything outside of flux-viewer.

Reply:
I'll leave this up to the POs to decide as a follow-up ticket.